examples/mxnet_mnist.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2017 Uber Technologies, Inc. All Rights Reserved.
```

Comment:
2017 => 2018 or 2019?

Reply:
Removed this license since pytorch also did not have it.